docs/src/tutorials/yocto-linux.md

```diff
@@ -0,0 +1,150 @@
+requirements:
+
+- 50 Gbytes of free disk space
+- a Linux distributions that supports the Yocto Project, see the [Supported Linux
+Distributions](https://docs.yoctoproject.org/4.0.4/ref-manual/system-requirements.html#supported-linux-distributions)
+section in the Yocto Project Reference Manual. For detailed information on preparing your build host, see the [Preparing
+the Build Host](https://docs.yoctoproject.org/4.0.4/dev-manual/start.html#preparing-the-build-host) section in the Yocto
+Project Development Tasks Manual.
+- Git 1.8.3.1 or greater
+- tar 1.28 or greater
+- Python 3.6.0 or greater.
+- gcc 5.0 or greater.
+
+For the purposes of the tutorial, we assume `~/yocto-thinedge` working directory. If using other directory, be sure to
+change paths where needed.
+
+### Build Host Packages
+
+Install essential packages:
+
+```bash
+$ sudo apt install gawk wget git diffstat unzip texinfo gcc build-essential chrpath socat cpio python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev pylint3 xterm python3-subunit mesa-common-dev zstd liblz4-tool
+```
+
+### Clone Poky Linux repository
+
+Clone the Poky Linux distribution sources. We'll be using version `kirkstone`. You can use `--depth=1` to speed up the
+process. If using these options, to see previous commits or other branches see
+[here](https://stackoverflow.com/questions/29270058/how-to-fetch-all-git-history-after-i-clone-the-repo-with-depth-1)
+and [here](https://stackoverflow.com/questions/17714159/how-do-i-undo-a-single-branch-clone).
+
+```
+$ git clone git://git.yoctoproject.org/poky --branch=kirkstone --depth=1
+```
+
+### Add layers
+
+We'll be using `meta-tedge` and `meta-openembedded`. First, fetch the repositories:
+
+```bash
+git clone https://github.com/thin-edge/meta-tedge
+git clone git://git.openembedded.org/meta-openembedded
+```
+
+Next, add the following layers:
+
+```bash
+bitbake-layers add-layer ~/yocto-thinedge/meta-openembedded/meta-oe
+bitbake-layers add-layer ~/yocto-thinedge/meta-openembedded/meta-python
+bitbake-layers add-layer ~/yocto-thinedge/meta-openembedded/meta-networking
+bitbake-layers add-layer ~/yocto-thinedge/meta-tedge
+
+```
+
+### Configure the build
+
+Activate `Systemd` as default init manager by adding following line to `local.conf`:
+
+```
+INIT_MANAGER="systemd"
+```
+
+Build `tedge` by running following command :
+
+```bash
+$ bitbake tedge-full-image
+```
+
+### Run the build
+
+When the build it complete, run it with qemu:
+
+```bash
+$ runqemu nographic
+```
+
+
+## Further recommendations
+
+After building the reference distribution and image, you can explore creating your own layer and image, and then
+integrating `tedge-*` recipes for it. 
+
+### Configure and run the layer on Raspberry Pi device
+
+> Note: following installation process will base on tutorial from the previous chapter
```

Comment:
The following installation process will be based on the tutorial from the previous chapter